### PR TITLE
Change a few monitoring scripts to use metric_sender

### PR DESCRIPTION
**Pull request by zgalor**
Use MetricSender instead of ZaggSender in monitoring scripts

depends on https://github.com/openshift/openshift-tools/pull/1747

### CODE REVIEW

**Comment by zgalor** [comment]
@twiest PTAL

**Comment by zgalor** [comment]
@joelddiaz PTAL

**Review by twiest — APPROVED** [review]
@zgalor This looks great!

This is ready to merge right after our holiday freeze is lifted.

**Pull request merged**
